tmt/steps/discover/shell.py

```diff
@@ -286,6 +286,23 @@ def go(self) -> None:
         assert self.workdir is not None
         testdir = self.workdir / "tests"
 
+        # for imported plans print details about a remote plan
+        parent = cast(Optional[tmt.steps.discover.Discover], self.parent)
+        if parent and parent.plan._original_plan and \
+                parent.plan._original_plan._remote_plan_fmf_id:
+
+            remote_plan_id = parent.plan._original_plan._remote_plan_fmf_id
+
+            # FIXME: cast() - https://github.com/python/mypy/issues/7981
+            # Note the missing Optional for values - to_minimal_dict() would
+            # not include unset keys, therefore all values should be valid.
+            for key, value in cast(Dict[str, str], remote_plan_id.to_minimal_spec()).items():
+
+                if key == 'ref' and remote_plan_id._ref_expanded:
+                    value = remote_plan_id._ref_expanded
+
+                self.info(f'imported-{key}', value, 'green')
```

Comment:
Hm, the problem with this is that 'import key' in English is an imperative and while it makes sense as an attribute name (telling `tmt` to import something) in logging it is not common to use imperatives but rather past tense or present continuous tense (since the imperative there looks confusing).

Reply:
Anyway, I will go with the proposed option in the next update.